po/meson.build

```diff
@@ -1 +0,0 @@
-i18n.gettext(gettext_package, preset: 'glib')
```

Comment:
This doesn't seem to be reimplemented but on the other hand it seems like the project never actually went and created those translations anyway?

Reply:
Yes, this was based on the gtk-rs meson template, so it wasn't using these things anyway.

Reply:
In that case, you might want to remove the unused po/ directory altogether?